src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="box">
+        <div className="display">
+          <Display />
+        </div>
+        <div className="button">
+          <ButtonPanel handleClick={this.handleClick} />
```

Comment:
- [ ] Where is this.handleClick coming from? 
Make sure all the functions you are used are declared somewhere.

- [ ] App should be a functional component:
<img width="821" alt="appFunctionalComponent" src="https://user-images.githubusercontent.com/16193212/93276848-1739ae80-f797-11ea-9ec3-507be934544b.png">